lnwallet/wallet.go

```diff
@@ -186,6 +185,13 @@ type addCounterPartySigsMsg struct {
 	// version of the commitment transaction.
 	theirCommitmentSig []byte
 
+	// This channel is used for the wallet to pass the completed channel
+	// back into the Funding Manager at the final stage in the funding
```

Comment:
The reason this is written with channeldb.OpenChannel rather than lnwallet.LightningChannel is that waitForFundingConfirmation uses a few fields from lnwallet.LightningChannel that aren't accessible to the FundingManager from outside the lnwallet package. FundingOutpoint, NumConfsRequired and IsPending are currently not accessible. I believe we could export LightningChannel.channelState, or we could add exported methods get those fields. Any preference for either of those solutions, leaving it as-is or some other solution?